src/TMxpMudlet.h

```diff
@@ -34,7 +34,7 @@ class TMediaData;
 
 class TMxpMudlet : public TMxpClient
 {
-    inline static const QString scmVersion = QStringLiteral("%1%2").arg(QString::fromLatin1(APP_VERSION), QString::fromLatin1(APP_BUILD));
+    inline static const QString scmVersion = QStringLiteral("%1%2").arg(QString::fromUtf8(APP_VERSION), QString::fromUtf8(APP_BUILD));
```

Comment:
As it happens we do not need to do such a convoluted concatenation, this can be simply done as:
```cpp
     inline static const QString scmVersion = QStringLiteral(APP_VERSION APP_BUILD);
```

Reply:
True enough, replacing.